Triangulation_3/include/CGAL/Regular_triangulation_3.h

```diff
@@ -166,12 +174,16 @@ namespace CGAL {
     using Tr_Base::find_conflicts;
     using Tr_Base::is_valid;
 
-    Regular_triangulation_3(const Gt & gt = Gt(), Lock_data_structure *lock_ds = NULL)
-      : Tr_Base(Tr_Base_Gt(gt), lock_ds), hidden_point_visitor(this)
+    Regular_triangulation_3()
+      : Tr_Base(Tr_Base_Gt(), NULL), hidden_point_visitor(this)
+    {}
+
+    Regular_triangulation_3(const Gt &, Lock_data_structure *lock_ds = NULL)
+      : Tr_Base(Tr_Base_Gt(), lock_ds), hidden_point_visitor(this)
```

Comment:
We then need to add two constructors to `Regular_triangulation_euclidean_traits_3`. I fixed that in commit 2a4a41a612f8da5f0130ff043f6d1298d4512504